docs/contributing/how-to-add-files-to-build-system.md

```diff
@@ -16,6 +16,12 @@ the `bakefile_gen` tool. Run it from `$(wx)/build/bakefiles` directory and it wi
 regenerate all outdated makefiles. See `$(wx)/build/bakefiles/README` for more
 details.
 
+You can also run Bakefile from a Docker or Podman container and avoid the need
+to install it (expanding "$(wx)" to be the path to wx install):
+
+    docker run --rm -v $(wx):$(wx) -w `pwd`
```

Comment:
Sorry, I don't understand the use of `-w` here: does the current working directory on the host really make sense as the working directory inside the container?